_features_versioneye/04_fixes.md

```diff
@@ -0,0 +1,18 @@
+---
+# Page title.
+title: Automated Fixes
```

Comment:
I would move this after 'Broad Ecosystem Support'

Reply:
done